src/TourManager/Repository/Configurations/ClientConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Travely.TourManager.Repository.Repositories.Entities;
+
+namespace Travely.TourManager.Repository.Configurations
+{
+    class ClientConfiguration : IEntityTypeConfiguration<Client>
```

Comment:
I'd prefer writing the visibility explicitly. internal class smth...., but your decision anyways.